config/locales/en.yml

```diff
@@ -151,3 +151,13 @@ en:
       caution_type:
         edit:
           page_title: Caution types
+    conviction:
```

Comment:
For exit / static or content pages, I've found out it is easier to maintain, produce and "read", if we just use translated views, instead of locales, in this case a `show.en.html.erb`, as the locales can grow a lot or be complicated with html markup sometimes. Also when the content is still WIP or likely to change.

But it is perfectly fine also this. It is just a suggestion.